__tests__/itinerary.test.js

```diff
@@ -0,0 +1,13 @@
+const Itinerary = require("../src/itinerary");
+
+describe("Itinerary", () => {
+  it("returns an object", () => {
+    expect(new Itinerary()).toBeInstanceOf(Object);
+  });
+  it("sets the ports property of the itinerary", () => {
+    const amsterdam = jest.fn();
+    const newcastle = jest.fn();
+    const itinerary = new Itinerary([amsterdam, newcastle]);
+    expect(itinerary.ports).toEqual([amsterdam, newcastle]);
```

Comment:
Its another convention to leave a clear line before the assertions start in each test to help with readability, should add one in here
```
    const itinerary = new Itinerary([amsterdam, newcastle]);

    expect(itinerary.ports).toEqual([amsterdam, newcastle]);
```